### ISSUE TITLE
Export gbXML bug for shading surfaces

### ISSUE THREAD

**Issue by mostaphaRoudsari** [NREL/OpenStudio#2422]
I opened the seb.osm file which comes with the installation, export it as gbXML and then import it back, and the shading surfaces are imported incorrectly.

![image](https://cloud.githubusercontent.com/assets/2915573/21535779/f3d4013e-cd47-11e6-9691-88712e010803.png)

![image](https://cloud.githubusercontent.com/assets/2915573/21535764/adf200a8-cd47-11e6-8332-eae1ab7a93b6.png)

I used OpenStudio 2.00 but also tested 1.13 C# library to import it and I get the same error.

**Comment by macumber** [comment]
Weird, is the top picture the original OSM and the bottom is after export to gbXML and import back to OSM?

**Comment by mostaphaRoudsari** [comment]
Yes. The one on the top is the osm file that comes with the installation and the one in the bottom is the xml file exported and imported back.

PS: I also couldn't get the schedules, loads, etc but I thought I keep them for when the issues that I have already sent is fixed or do you want me to post them right now?
